### ISSUE TITLE
Forwarding wire:model input Events does not work in livewire version 1.3.0

### ISSUE THREAD

**Issue by isaackearl** [livewire/livewire#1236]
### Description
I discovered after upgrading one of my projects that my javascript integrations that use forwarding of wire:model input events stopped working.  I decided to investigate in a clean project just to be sure, and was able to confirm that even the simplest examples don't seem to work in Livewire 1.3.0.  If I downgrade to 1.2.0 then everything starts working again.

### Steps to reproduce

here is the code sample I used to test this in a new project

```
<div id="main">
    <div wire:model="count">
        Count:
        <button id="button"
                type="button"
                x-data="{ count: 0 }"
                @click="count++;$dispatch('input', count)"
                x-text="count"/>
    </div>
    <div>
        Livewire Count: {{ $count }}
    </div>
</div>
```
Using livewire 1.3.0 this doesn't work.  Downgrade to 1.2.0 and it starts working.  Upgrade back to 1.3.0 and it stops working again.  

You can see in the screenshot below that livewire still detects the input event but the value that is sent through for the count is always an empty string.
<img width="1025" alt="Screen Shot 2020-07-08 at 12 28 02 AM" src="https://user-images.githubusercontent.com/9354830/86876151-0d6a3f80-c0b2-11ea-8da9-3f04faa53e01.png">

**Context**
- Livewire version: 1.3.0
- Laravel version: 7.19.0
- Browser: Chrome

**Comment by mblarsen** [comment]
I think this is what I'm running into as well here PR #1226 (see comments).

**Comment by zepfietje** [comment]
I'm running into the exact same problem and can confirm the described behavior.
Downgrading to 1.2.0 indeed fixes the issue as a workaround.

**Comment by calebporzio** [comment]
This should be fixed in 1.3.1

Thanks @mblarsen for the PR!
